src/VisualStudio/Core/Next/FindReferences/StreamingFindReferencesPresenter.TableDataSourceFindReferencesContext.cs

```diff
@@ -580,8 +580,16 @@ private RoslynDefinitionBucket GetOrCreateDefinitionBucket(DefinitionItem defini
 
             public override Task ReportProgressAsync(int current, int maximum)
             {
-                //var progress = maximum == 0 ? 0 : ((double)current / maximum);
-                // _findReferencesWindow.SetProgress(current, maximum);
+                try
+                {
+                    // The original FAR window exposed a SetProgress(double). Ensure that we 
+                    // don't crash if this code is running on a machine without the new API.
+                    _findReferencesWindow.SetProgress(current, maximum);
+                }
+                catch
+                {
```

Comment:
Include a comment about why this is here, and a bug to remove it once we drop support for builds where it doesn't work.